src/components/GameBoard.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import styled from "styled-components";
+import HeartRow from "./HeartRow";
+import { useHistory } from "react-router-dom";
+import WinLossWindow from "../components/WinLossWindow";
+import drawGameState from "../GameData/Draw";
+import createEvents, { handleEvents } from "../GameData/Events";
+
+const StyledCanvas = styled.canvas`
+  background: ${props => props.theme.accent};
+`;
+
+const GameContainer = styled.section`
+  margin: 0;
+  padding: 0;
+  position: relative;
+`;
+
+const Modal = styled.dialog`
+  border: none;
+  padding: 0;
```

Comment:
again: is padding: 0 not set by default?

Reply:
Unfortunately not on the dialog tage :(